debrisframe/c1Ti/c1TiCfg.ini

```diff
@@ -0,0 +1,234 @@
+### Config File - This file contains the main settings for the debris flow simulation run
+## Copy to  local_c1TiCfg.ini and set your parameters
+
+
+[GENERAL]
+
+
+[com1DFA_com1DFA_override]
```

Comment:
To keep it a bit more simple, I suggest to really only add those parameters that actually differ from the default com1dfa. And not those that "might" be changed....